dspace-api/src/main/java/org/dspace/authenticate/ShibAuthentication.java

```diff
@@ -279,13 +279,6 @@ public int authenticate(Context context, String username, String password,
 	public int[] getSpecialGroups(Context context, HttpServletRequest request)
 	{
 		try {
-			// User has not successfuly authenticated via shibboleth.
-			if ( request == null || 
-					context.getCurrentUser() == null || 
-					request.getSession().getAttribute("shib.authenticated") == null ) {
-				return new int[0];
-			}
```

Comment:
I have removed this section because it prevented in my case the whole block from being executed. I guess it was because the request is not set correctly in my environment, but I'll investigate that again.